ts.nu

```diff
@@ -86,3 +84,23 @@ export def "ts parse" [    # -> string
 export def "ts config" [] {
     tree-sitter init-config
 }
+
+# Regenerates tree-sitter bindings and generated manifests
+export def "ts regen" [] {
```

Comment:
This is handy, thanks! Does it cause any trouble if we don't remove the existing ones BTW? I thought (probably wrong) `--update` means incremental changes.

Reply:
> I think one problem is that there can be "stuck" patterns that newer versions of `tree-sitter-cli` leave alone 

True, a whitelist of files we want to keep?

BTW, tree-sitter-cli 0.26.0 just got released, would you like to run regen with that?

Reply:
> BTW, tree-sitter-cli 0.26.0 just got released, would you like to run regen with that?

Looks like the NPM packages aren't up to date yet (so node tests might break in CI):
https://www.npmjs.com/package/tree-sitter

we can wait for those or do that as a followup.

Reply:
this was the output of running `tree-sitter test -r`:
<img width="337" height="825" alt="image" src="https://github.com/user-attachments/assets/1fbafadc-ae85-4c34-90f8-9e3beecad129" />


The issue was fixed after running `cargo install tree-sitter-cli` _without_ making any changes to `grammar.js`.